android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageBase.java

```diff
@@ -351,7 +350,7 @@ protected String decryptBytes(@NonNull final Key key, @NonNull final byte[] byte
     throws GeneralSecurityException, IOException {
     final Cipher cipher = getCachedInstance();
 
-    // decrypt the bytes using a CipherInputStream
+    // decrypt the bytes using cipher.doFinal()
     try (ByteArrayInputStream in = new ByteArrayInputStream(bytes);
          ByteArrayOutputStream output = new ByteArrayOutputStream()) {
```

Comment:
```suggestion
    try {
```

I don't think the input stream and output stream are really used anymore, since the decrypt operation now works with byte arrays. These should probably be removed.

Reply:
👍 This was a miss. I should've cleaned this up.